src/assets/i18n/global/en-global.json

```diff
@@ -327,5 +327,6 @@
     "SUCCESSFULLY_REGISTERED": "Successfully registered!",
     "GO_TO_WALLET": "Go to Login",
     "ENTER_VALID_AADHAAR_NUMBER": "Enter valid Aadhaar number",
-    "GENDER_REQUIRED": "Gender is required"
+    "GENDER_REQUIRED": "Gender is required",
+    "LOGIN_VIA_MERI_PEHCHAAN":"Login via Meri Pehchaan"
```

Comment:
@shreyas1434shinde text should be login or login via meri pehchaan

Reply:
@itsvick comment resolved